### ISSUE TITLE
Distributed Query History

### ISSUE THREAD

**Issue by jalseth** [kolide/fleet#2068]
It would be nice if the Fleet UI maintained a history of recent distributed queries run by the user to make writing and iterating on queries easier.

This history doesn't need to be stored on the server (although I believe these are already stored in the `queries` table with a `distributed_` prefix on the `name` column), it's fine if the history is lost by closing the tab/window.

**Comment by zwass** [comment]
Good idea! This should not be too hard to do, but we'll need to figure out how to display it in the UI. Thoughts?

**Comment by jalseth** [comment]
I'd like it to be a [collapsable accordian](https://getbootstrap.com/docs/4.1/components/collapse/#accordion-example) with a table listing the query and datetime, probably located below the description box. It should start collapsed as well, only showing the history if the user clicks on it. Also, clicking on an entry in the table should populate the query into the SQL input box.
